notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>I tried to run it and got error:  <code>OSError: libtorch_cuda.so: cannot open shared object file: No such file or directory,</code> Did you face with the same ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Tested the notebook before PR. Did not face this error.

Reply:
@sbalandi  probably you have torch installed without cuda.
@spradeep28  does this notebook has dependency on cuda? Can we avoid this?

Reply:
Updated pip install with cpu wheel.